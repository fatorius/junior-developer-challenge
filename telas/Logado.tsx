import VALUES from "@/VALUES";
import { View, Text } from "react-native";

function Logado() {
	return (
		<View
			style={{
				flex: 1,
				justifyContent: "center",
				alignItems: "center",
			}}>
			<Text
				style={{
					fontSize: VALUES.TamanhoFonte.xLarge,
					color: VALUES.Tema.corPrimaria,
					fontFamily: VALUES.Fontes.negrito,
					textAlign: "center",
				}}>
				Bem vindo! Voce est√° logado
			</Text>
		</View>
	);
}

export default Logado;
