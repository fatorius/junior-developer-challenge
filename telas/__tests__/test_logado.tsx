import React from "react";
import renderer from "react-test-renderer";
import Logado from "../Logado";

test("Render: Logado", () => {
	const tree = renderer.create(<Logado />).toJSON();
	expect(tree).toMatchSnapshot();
});
