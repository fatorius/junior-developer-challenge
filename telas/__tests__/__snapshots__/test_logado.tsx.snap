// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render: Logado 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      {
        "color": "#1F41BB",
        "fontFamily": "poppins-bold",
        "fontSize": 65.01635745034561,
        "textAlign": "center",
      }
    }
  >
    Bem vindo! Voce est√° logado
  </Text>
</View>
`;
